glotpress.php

```diff
@@ -209,3 +209,14 @@ function gp_deactivate_plugin( $network_wide ) {
 
 }
 register_deactivation_hook( GP_PLUGIN_FILE, 'gp_deactivate_plugin' );
+
+add_action(
```

Comment:
As we remove these assets (styles and JavaScript), why do you enqueue them here?

Reply:
This is to enqueue the scripts and styles when you have the tour plugin installed. For some weird reasons, having the tour plugin installed along with GlotPress doesn't enqueue these scripts introduced by the tour plugin.

Reply:
I've removed these lines and in preparation to close the 4.0 milestone, I'll suggest we fix the tour issue in another PR.